salsa_dancing_molecules/simulations/nve.py

```diff
@@ -35,16 +37,26 @@ def run_for_materials(formula, api_key, steps, output_path, repeat=0):
         if repeat > 0:
             atoms = atoms.repeat(repeat)
 
-        try:
-            print(f'Simulating for {symbols}')
-            run(atoms, steps, output_name)
-        except AsapError as e:
-            print(f'ASAP3 error for {symbols}: {e}')
-            print('Trying without ASAP3...')
-            run(atoms, steps, output_name, False)
-
-
-def run(atoms, steps, output_path, use_asap=True):
+        while True:
+            try:
+                print(f'Simulating for {symbols}')
+                run(atoms, steps, output_name, potential)
+            except ValueError as e:
```

Comment:
What if this loops forever?

Reply:
I don't think that is possible, the ValueError will only be generated if the cell size is too small and increasing the size enough times should solve the problem. There might exist some weird edge case if the cell is wrongly designed (i.e. a cell with size 0) or if the potential is wrongly designed, but I don't think OpenKIM potential will have that problem.